eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -222,6 +226,15 @@ private Collection<Dependency> deps() {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Check if dependency is not needed at runtime.
+     * @param dep Maven dependency
+     * @return True if it's not needed at runtime
+     */
+    private static boolean isNotRuntimeRequired(final Dependency dep) {
+        return !dep.getScope().contains("test") && !dep.getScope().contains("provided");
```

Comment:
@volodya-lombrozo by the way, why do we use `contains` instead of `equals`? They are two strings to compare.

Reply:
@yegor256 done